src/ontology/cl-edit.owl

```diff
@@ -27635,6 +27637,14 @@ SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P60041))
 
+# Class: obo:CL_4023026 (direct medium spiny neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Medium_spiny_neuron&oldid=1109726850"^^xsd:string) obo:IAO_0000115 obo:CL_4023026 "A medium spiny neuron that projects to the internal division of the globus pallidus (GPi) or the substantia nigra pars reticula (SNpr or SNr)"^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023026 <https://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30096299"^^xsd:string) Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasExactSynonym obo:CL_4023026 "dSPN"^^xsd:string)
+AnnotationAssertion(rdfs:label obo:CL_4023026 "direct medium spiny neuron")
```

Comment:
I believe it is more accurate to label this cell type 'direct pathway medium spiny neuron' since "direct" describes the pathway in which the neurone is located and not a property of the neurone itself. Additionally 'direct pathway medium spiny medium neuron' is what is used in the references provided.

Reply:
will change it and make the current label a synonym